app/src/main/java/de/danoeh/antennapod/adapter/FeedItemlistAdapter.java

```diff
@@ -153,7 +141,9 @@ public View getView(final int position, View convertView, ViewGroup parent) {
                     holder.episodeProgress.setProgress(((ItemAccess) itemAccess).getItemDownloadProgressPercent(item));
                     holder.published.setVisibility(View.GONE);
                 } else {
-                    holder.episodeProgress.setVisibility(View.GONE);
+                    if(media.getPosition() == 0) {
```

Comment:
I'm not 100% sure if this actually was my intention, but I like seeing a bar for paused episodes.
This way, you can scroll all episodes and find ones that you started but never finished just by looking.